### PR TITLE
Fix/stake pool validation

### PR DESCRIPTION
**Pull request by dabasov**

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/stake-pool-validation  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | fix/stake-pool-validation  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by Kishan-Dhakan** [comment]
It's a duplicate of https://github.com/0chain/system_test/pull/558. Anyway, I'll close https://github.com/0chain/system_test/pull/558.

File: tests/cli_tests/zwalletcli_miner_stake_test.go [hunk 1]
```diff
@@ -341,10 +339,9 @@ func TestMinerStake(testSetup *testing.T) {
 			"tokens":   intToZCN(max_stake)/2 + 1,
 		}), true)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zwalletcli_miner_stake_test.go:340]: Change to false since we expect failure so no need to retry

**Review by Kishan-Dhakan — APPROVED** [review]
Minor change requested

**Pull request merged**